Add Rancher Desktop tutorial (#847)
epinio/epinio: docs/user/tutorials/install_epinio_on_rancher_desktop.md

# Rancher Desktop configuration

## Rancher Desktop Prerequisites

* Running on Windows requires Windows Subsystem for Linux (WSL) which is automatically installed by Rancher Desktop
* Epinio v0.1.1 has been tested with Rancher Desktop incl. kubernetes version v1.21.5

### Install Rancher Desktop

Install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) from Rancher Desktop for your operating system

## Setup Kubernetes

> When running Rancher Desktop for the first time wait until the initialization is completed. Make sure that a supported Kubernetes version is selected under `Kubernetes Settings`, e.g. **v1.21.5**

Author: I was thinking about it, the problem is that the higher version 1.22.2 does not work atm because of #833. Should we maintain a document were we list the supported Kubernetes versions and reference this here to reduce duplication?

> When running Rancher Desktop for the first time wait until the initialization is completed. Make sure that a supported Kubernetes version is selected under `Kubernetes Settings`, e.g. **v1.21.5**

Reviewer: I think "versions newer than" is ok. "versions up to" is also ok. But cherry-picked versions is not. If something doesn't work we should probably know why unless it's a too-old version and nobody cares anymore.

## Install epinio

Make sure Rancher Desktop is running

### Windows

> 1. Start a terminal (e.g. type `cmd` in the search field) and change to the directory where `epinio-windows-amd64` is located, e.g. `cd Downloads`

Reviewer 2: Would it be worth a brief explanation of how PATH works in windows and suggest a location to move the binary to?

Reviewer 3: IMHO yes

# Rancher Desktop configuration

## Rancher Desktop Prerequisites

* Running on Windows requires Windows Subsystem for Linux (WSL) which is automatically installed by Rancher Desktop

### Install Rancher Desktop

Install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) from Rancher Desktop for your operating system

## Setup Kubernetes

When running Rancher Desktop for the first time wait until the initialization is completed. Make sure that a supported Kubernetes version is selected under `Kubernetes Settings`, e.g. **v1.21.5**

## Install epinio

Make sure Rancher Desktop is running.

> Since there might be cases where Rancher Desktop reports Kubernetes as running but some pods are not yet ready the state can be manually verified by executing in a terminal `kubectl get pods -A` and veryfing that all pods report the status `Running` or `Completed`.

Reviewer 3: ```suggestion
Rancher Desktop can report Kubernetes as running while some pods are actually not yet ready.
Manual verification is possible by executing the command `kubectl get pods -A` in a terminal and checking that all pods report either `Running` or `Completed` as their status.
```

I prefer shorter sentences, with a more active voice.

### Windows

1. Start a terminal (e.g. type `cmd` in the search field) and change to the directory where `epinio-windows-amd64` is located, e.g. `cd Downloads`

2. Run `epinion-windows-amd64 install`

> 3. To allow Epinio being executed directly from any terminal copy the binary to a directory of your choice and add it to the `PATH` variable as described [here](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)

Reviewer 3: ```suggestion
3. To allow Epinio being executed directly from any terminal copy the binary to a directory of your choice and add it to the `PATH` variable as described by [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)
```

Attribution, and IMHO nicer phrasing. I try to avoid `here` links.

Reviewer 3: ```suggestion
3. Copy the binary to a directory of your choice and add it to the `PATH` variable as described by [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53). This allows execution of Epinio directly from any terminal.
```

As the first suggesstion, and split the long sentence.
